### DISCUSSION TITLE
Github codespace changes when a student inside the class.

### DISCUSSION THREAD

**Discussion by pratikdevelop** [community/community#101543] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

As per github doc, when we used codespace with classroom should charge the organization for other, not for student. 

This is misconfigured issue?

**Comment by mayankzarkar** [comment, marked as answer]
For the org. I created for the classroom, the default was for codespace ownership to be "user" not "organization". You can chagne redio button.

![image](https://github.com/community/community/assets/123540238/71b31a2c-010b-470c-b7c2-be6e5ab66801)
